### DISCUSSION TITLE
Power Reading and Calibration

### DISCUSSION THREAD

**Discussion by Deewian** [emporia-vue-local/esphome#185] [Q&A] [answered]
Is there a different way to calculate or calibrate individual circuit power?

I used the 0.022 factor to get calibration voltage on Phase A and B and it's now reading the correct data.

On circuit 1, I read 0.55A on my meter but the integration says 16.7W of power. Using the P=V*I, I would expect to have about 135.3W of power with a voltage reading of 123V and a multiplier of 2 since it's 240V. I would need to change the multiplier to 8.1 which I find very high and far from reality.

Other thing I noticed is sometimes the power on circuit 1 goes higher than power on Phase A and B.

![image](https://github.com/emporia-vue-local/esphome/assets/50123435/8291d392-4318-43b6-9c38-5b5651332022)

I tried replacing the emporia CT clamp for a different one, I checked to make sure they were all well connected and that phase voltage is the right one but I get the same result.

Thank You

**Comment by flaviut** [comment]
This is strange, a multiplier of 8.1 is crazy. Generally these multipliers should be either 1 or 2.

Could the phases set up in the wrong order? What kind of system do you have/where do you live? (3 phase, split phase?)

You can also try removing the `filters: [...]` sections to maybe get some more hints about what's wrong. I'm not really sure.

**Reply by Deewian** [reply, marked as answer]
I'm in Canada, I have a two phase 120/240 panel.

I have main probes on A and B input of the emporia device. I tried removing the filter for circuit 1 and value reads the same but updated a lot faster which is expected.

I tried reversing red and black wires based on another discussion that said newer devices can have A and B reversed but all readings drop to 0w when I do that.

Only thing I can see is that load might be too low for the CT to read precisely. Other circuits pulling more power give better results and my mains are giving good info after calibration since I have only 0.10Kw/h daily consumption difference with the power company.

Just did the test as I was writing this reply and with  2A on the circuit, the value is good so I guess it really comes down to losing precision if amps are too low and I don't think we can do much about this.

Thank You

**Reply by flaviut** [reply]
I would expect 0.5A to be read accurately, but I'm glad this doesn't add up to very much error in the end. :+1:
